src/components/battery/BatteryController.cpp

```diff
@@ -61,12 +61,12 @@ void Battery::SaadcInit() {
 }
 
 void Battery::SaadcEventHandler(nrfx_saadc_evt_t const* p_event) {
-  static const Utility::LinearApproximation<uint16_t, uint8_t, 6> aprox {{{
-    {3500, 0},  // Minimum voltage before shutdown (depends on the battery)
-    {3600, 10}, // Keen point that corresponds to 10%
-    {3700, 25},
-    {3750, 50},
-    {3900, 75},
+  static const Utility::LinearApproximation<uint16_t, uint8_t, 6> approx {{{
+    {3500, 0}, // Minimum voltage before shutdown (depends on the battery)
+    {3616, 3},
+    {3723, 22},
+    {3776, 48},
+    {3979, 79},
     {4180, 100} // Maximum voltage during charging is 4.21V
```

Comment:
Let's remove the comments, because they're not accurate.

Reply:
I agree about the first one, but I think it's useful to have some info about the max charge voltage being different than the voltage at 100% when not charging.

During charging, I saw 4.20V on one PT, 4.21V on another, and then 4.25V on my 3rd.

Reply:
You can add a comment outside of the values that says the voltage is higher during charging, although it's normal behaviour for battery charging.